src/feed.coffee

```diff
@@ -52,19 +45,24 @@ class EditorialFeed
           (file) =>
             $.get(file, (
               (data) =>
-                images = @_parse data
-                if images? and images.length > 0
-                  console.log "Replacing images #{@images.length} -> #{images.length}"
-                  @images = images
+                imageUrls = @_parse data
+                if imageUrls? and imageUrls.length > 0
+                  console.log "Replacing images #{@images.length} -> #{imageUrls.length}"
+                  promises = (@_cache(url) for url in imageUrls)
```

Comment:
This is going to result in a lot of requests on app start up. I think it will be better if we add some delay between each request. For low power players and players with bad connectivity I can see this causing problems.